demo/collapsible/showmore/playground

```diff
@@ -0,0 +1,113 @@
+const Table = () => (
+  <table width="100%">
+    <thead>
+      <tr>
+        <th align="left">Component</th>
+        <th align="left">Type</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>backToTop</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>badge</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>button</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>card</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>checkbox</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>helpText</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>icon</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>image</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>input</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>label</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>panel</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>popover</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>progressBar</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>radioButton</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>slider</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>spinner</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>switch</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>tag</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>textarea</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>toast</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>tooltip</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>upload</td>
+        <td>Atom</td>
+      </tr>
+      <tr>
+        <td>validationText</td>
+        <td>Atom</td>
+      </tr>
+    </tbody>
+  </table>
+)
+const children = <Table />
+const copy = 'Ver más'
+const maxHeight = 210
+
+return (<CollapsibleShowmore 
+          children={children}
```

Comment:
First time I see this. Just curious, is it the same as `<Component><Table/></Component>` isn't it?

Reply:
Yes. It probably makes more sense when the `CHILDREN` const is made up of multiple components though.

Reply:
I want pass a component as a children, and in this demo I created a simple Table component.

Reply:
```
<CollapsibleShowmore copy={copy} maxHeight={maxHeight}>
  {children}
</CollapsibleShowmore>
```